app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -617,5 +620,12 @@ public void backButtonClicked() {
     public MediaDetailPagerFragment getMediaDetailPagerFragment() {
         return mediaDetailPagerFragment;
     }
+
+    // click listener to toggle description
+    private OnClickListener toggleDescriptionListener = view -> {
+      if(view.getVisibility() == View.GONE) {
```

Comment:
We use 4-spaces indentation.

Reply:
Also, space after `if` per https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace